frontend/src/views/applications/SelectFacilityView.vue

```diff
@@ -8,8 +8,17 @@
         <span>Please review the following pre-populated information for correctness and contact your organization's account manager to make updates if required.</span>
       </div>
       <OrganizationInfo :loading="loading" :organization="organization" />
+      <div v-if="organization?.businessTypeCode === BUSINESS_TYPE_CODES.NON_PROFIT_SOCIETY" class="mb-6">
```

Comment:
I don't think we need a div here. we can move v-if to NotForProfitQuestions.

Reply:
done